extensions/EXT/EXT_texture_format_BGRA8888.txt

```diff
@@ -135,12 +141,35 @@ Interactions with the OpenGL ES 2.0 specification
     and the number of bits each format contains for color (R, G, B, A),
     depth (D), and stencil (S) components:
     
-    |-----------------|------------------|------|------|------|------|------|------|
-    | Sized           | Renderable       | R    | G    | B    | A    | D    | S    |
-    | Internal Format | Type             | bits | bits | bits | bits | bits | bits |
-    |-----------------|------------------|------|------|------|------|------|------|
-    | GL_BGRA_EXT     | color-renderable | 8    | 8    | 8    | 8    |      |      |
-    |-----------------|------------------|------|------|------|------|------|------|
+    Sized           Renderable        R    G    B    A    D    S
+    Internal Format Type              bits bits bits bits bits bits
+    --------------- ----------        ---- ---- ---- ---- ---- ----
+    BGRA_EXT        color-renderable  8    8    8    8
```

Comment:
@fooishbar pointed out to me elsewhere that it would be *really* convenient if `BGRA8_EXT` was still added here, because that makes things more consistent for `glRenderbufferStorage()`, which makes format handling in Weston a lot cleaner. So I think I'll add that as well, if people don't mind. After all, that's what I did at first, so I assume people are happy with it...

Reply:
Yeah, I asked for it because it seemed very incongruous (and a real footgun) that BGRA_EXT would be an acceptable sized internal format but BGRA8_EXT would not.